doc/api/inspector.md

```diff
@@ -0,0 +1,64 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: v8.0.0
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+At any given moment there may be only one connected session (created either
+with this API or by connecting to an inspector socket). Creating a second
+session using this API will throw an exception.
+
+Up-to-date version of the V8 inspector protocol is published on the
+[protocol viewer][protocol-viewer]
+
+### Event 'notification'
+<!-- YAML
+added: v8.0.0
+-->
+
+* {Object} The notification
+
+Emitted when a notification from an inspector backend was received. It is also
+possible to subscribe to a specific inspector notification type by specifying
+the method. E.g. in this snippet the `debuggerPausedCallback` will be called
+whenever the program execution is suspended:
+```js
+session.on('Debugger.paused', debuggerPausedCallback);
```

Comment:
A link to https://chromedevtools.github.io/debugger-protocol-viewer/v8/Debugger/#event-paused can be helpful here, and an example showing the arguments `debuggerPausedCallback` should make it clearer too. If I'm understanding correct, it should be something like

```js
session.on('Debugger.paused', ({ callFrames, reason }) => {
  // ...
});
```

right?